libcrux-ml-dsa/src/ml_dsa_generic.rs

```diff
@@ -270,16 +261,16 @@ pub(crate) fn sign<
 
     let mut attempt = 0;
 
+    // TODO: This loop will have to be rewritten to go through hax.
```

Comment:
Can you collect these things in an issue for the verification?

Reply:
Opened https://github.com/cryspen/libcrux/issues/341 and referenced it in the comments.